params/config_extra.go

```diff
@@ -173,6 +176,25 @@ func GetChainConfig(agoUpgrade upgrade.Config, chainID *big.Int) *ChainConfig {
 		MuirGlacierBlock:    big.NewInt(0),
 		NetworkUpgrades:     getNetworkUpgrades(agoUpgrade),
 	}
+	if AvalancheFujiChainID.Cmp(c.ChainID) == 0 {
+		c.BerlinBlock = big.NewInt(184985) // https://testnet.snowtrace.io/block/184985?chainid=43113, AP2 activation block
+		c.LondonBlock = big.NewInt(805078) // https://testnet.snowtrace.io/block/805078?chainid=43113, AP3 activation block
+	} else if AvalancheMainnetChainID.Cmp(c.ChainID) == 0 {
+		c.BerlinBlock = big.NewInt(1640340) // https://snowtrace.io/block/1640340?chainid=43114, AP2 activation block
+		c.LondonBlock = big.NewInt(3308552) // https://snowtrace.io/block/3308552?chainid=43114, AP3 activation block
+	} else {
+		// In testing or local networks, we only support enabling Berlin and London prior
+		// to the initially active time. This is likely to correspond to an intended block
+		// number of 0 as well.
+		initiallyActive := uint64(upgrade.InitiallyActiveTime.Unix())
```

Comment:
this is mostly because `params.GetChainConfig` is called from e2e tests (avalanchego) and specifies `initiallyActiveTime` for the genesis